admin/js/jStorage.github.js

```diff
@@ -300,10 +300,8 @@
             var self = this;
             addr = "https://api.github.com/repos/" + this._config.repo + "/contents/" + name;
             githubRequest("GET", addr, self._config.token, false, function () {
-                console.log('listA', arguments);
                 if (arguments.length >= 2) {
                     var info = arguments[1];
-                    console.log('list', info);
                     if (info.type != "file") {
```

Comment:
Generally, conditionals can often be broken out into methods with descriptive names. In this case it looks innocent and quite small, but reading conditionals is a hurdle for someone new to understand the codebase.